src/screens/Messages/Conversation/MessagesList.tsx

```diff
@@ -340,18 +341,20 @@ export function MessagesList({
         />
       </ScrollProvider>
       <KeyboardStickyView offset={{closed: -bottomOffset, opened: 0}}>
-        {!blocked ? (
+        {convoState.status === ConvoStatus.Disabled ? (
+          <ChatDisabled />
+        ) : blocked ? (
+          footer
+        ) : (
           <>
-            {convoState.status === ConvoStatus.Disabled ? (
-              <ChatDisabled />
-            ) : (
-              <MessageInput onSendMessage={onSendMessage} />
+            {isConvoActive(convoState) && convoState.items.length === 0 && (
+              <ChatEmptyPill />
             )}
+            <MessageInput onSendMessage={onSendMessage} />
           </>
-        ) : (
-          footer
         )}
       </KeyboardStickyView>
+
```

Comment:
Moved everything into `KeyboardStickyView` so that the pill follows the keyboard up.